network-api/networkapi/wagtailpages/pagemodels/research_hub/relations.py

```diff
@@ -65,3 +65,23 @@ class ResearchDetailPageResearchTopicRelation(wagtail_models.TranslatableMixin,
     panels = [
         snippet_handlers.SnippetChooserPanel('research_topic'),
     ]
+
+
+class ResearchLandingPageFeaturedResearchTopicRelation(wagtail_models.TranslatableMixin, wagtail_models.Orderable):
+    research_detail_page = cluster_fields.ParentalKey(
```

Comment:
**Issue:** This attribute needs to be renamed to match the actually linked page type. 
```suggestion
    research_landing_page = cluster_fields.ParentalKey(
```

Reply:
Updated!